README.md

```diff
@@ -38,6 +38,7 @@ Currently, Deer supports these languages :earth_americas:. If you would like to
 * العربية :egypt:
 * Español :es:
 * Русский :ru:
+* Português do Brasil :pt_br:
```

Comment:
This emoji is broken. I'll approve this and fix it later.

Reply:
Any updated here @enzo-beltrami